oUF.xml

```diff
@@ -44,7 +44,7 @@
 	<!--@end-retail@-->
 
 	<!--@non-retail@-->
-	<Script file="elements\combopoints.lua"/>
+	<Script file="elements\classpowerclassic.lua"/>
```

Comment:
If this is the classic branch, why not just name it "classpower"?

Reply:
Both files are present in the development version.

The idea is to use the branch to see how long we (I) can reasonably easy keep API parity between retail and classic (and other old expansions if they release it) since we had the discussion internally and this is the main concern and the reason to go with separate branches. Also the branch can be used to create an embedded oUF version supporting both retail and classic for as long as we manage to keep this parity.